### PR TITLE
react-table: Correct RowInfo documentation and add original property.

### PR DESCRIPTION
**Pull request by ProTip**
If changing an existing definition:
- [ x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/react-tools/react-table/tree/v6.7.0#renderers
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by uniqueiniquity** [comment]
@royxue @psakalo @Havret Could you review this change?

File: types/react-table/index.d.ts [hunk 1]
```diff
@@ -601,9 +601,12 @@ export interface RowRenderProps extends Partial<RowInfo> {
 }
 
 export interface RowInfo {
-    /** Original row from your data */
+    /** Materialized row of data */
     row: any;
```
Comment (armanio123) [inline types/react-table/index.d.ts]: I believe the documenation states this is of type object.

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: types/react-table/index.d.ts (hunk 1 above)
Comment (ProTip) [reply types/react-table/index.d.ts]: This PR is not addressing the underlying type.

Comment (armanio123) [reply types/react-table/index.d.ts]: Sounds good to me, you can ignore this comment.

**Review by armanio123 — APPROVED** [review]

**Comment by yuliaHope** [comment]
@ProTip  it's broken after your changes. 
image

**Comment by ProTip** [comment]
This is due to a merge conflict introduced after my PR was opened. Unfortunately I do not have the permissions to correct this, and it took ~16 days for my last PR to be merged :|

**Comment by uniqueiniquity** [comment]
@ProTip If you have a fix, we can get it merged much quicker this time.

**Pull request merged**